README.md

```diff
@@ -89,6 +89,18 @@ Inquire if the type of local authentication policy is supported on this device w
 
 Get what type of hardware biometry support the device has. Resolves to a `Keychain.BIOMETRY_TYPE` value when supported, otherwise `null`.
 
+### `getSecurityLevel()`
+
+Get security level that is supported on the current device with the current OS.
+
+### Security Levels (Android only)
+
+If set, `securityLevel` parameter specifies minimum security level that the encryption key storage should guarantee for storing credentials to succeed.
+
+* `ANY` - no security guarantees needed (default value); Credentials can be stored in FB Secure Storage;
```

Comment:
can we export these as constants from the native module? (see `getConstants` in [the docs](https://facebook.github.io/react-native/docs/native-modules-android#the-toast-module)

Reply:
yep, I did that. It didn't simplify the code that much though, mostly because it exports them as globals in the module space, and all of the existing code is based on `Object`s:

what `getConstants` export is: `SECURITY_LEVEL_ANY` in the js scope, where we have `SECURITY_LEVEL.ANY` now.

So what I did is I wrapped these constants into an object. It is a slightly better code, I think, because the values of the constants aren't hardcoded twice, but it is not as huge of a win.